### ISSUE TITLE
bytes fields are missing in base objects

### ISSUE THREAD

**Issue by webmaster128** [stephenh/ts-proto#237]
For the proto message

```proto
message MsgInstantiateContractResponse {
  // Address is the bech32 address of the new contract instance.
  string address = 1;
  // Data contains base64-encoded bytes to returned from the contract
  bytes data = 2;
}
```

The following base bessage is generated:

```ts
const baseMsgInstantiateContractResponse: object = { address: "" };
```

Since the `.decode` method only overrides field numbers that are serialized, the `data` field is not set when it is not serialized.

E.g. the serialization `0a2b7761736d317674796d7a6b6b65753777387578666e687961676a6b6b70653278676a303776396371707574`:

```sh
$ echo 0a2b7761736d317674796d7a6b6b65753777387578666e687961676a6b6b70653278676a303776396371707574 | xxd -r -p | protoc --decode_raw
1: "wasm1vtymzkkeu7w8uxfnhyagjkkpe2xgj07v9cqput"
```

results in the object

```js
{
  address: 'wasm1vtymzkkeu7w8uxfnhyagjkkpe2xgj07v9cqput',
}
```

instead of the expected

```js
{
  address: 'wasm1vtymzkkeu7w8uxfnhyagjkkpe2xgj07v9cqput',
  data: new Uint8Array(),
}
```

which is defined in the interface

```ts
export interface MsgInstantiateContractResponse {
  /** Address is the bech32 address of the new contract instance. */
  address: string;
  /** Data contains base64-encoded bytes to returned from the contract */
  data: Uint8Array;
}
```

**Comment by ethanfrey** [comment]
Would it be better to return `data: new Uint8Array()` or to change the type to `data?: Uint8Array`?

I assume the former is desired and less breaking, but it might be good to clarify what is wanted. I guess in general the default values should be added to the object for any primiative field that is not serialized (number, Long, string, Uint8Array). Is the the correct desired behavior?

**Comment by webmaster128** [comment]
This is a `bytes` specific issues caused by this check: https://github.com/stephenh/ts-proto/blob/v1.73.0/src/main.ts#L524. I dound this should be there.

We don't want `data?: Uint8Array` because the proto document does not differentiate between unset and empty semantically. They must be deserialized into the same thing.

**Comment by stephenh** [comment]
This is released in v1.77.0. Thanks!
